### ISSUE TITLE
not work on Windows?

### ISSUE THREAD

**Issue by canfoderiskii** [pl-ca/CtagsFuncComplete#1]
Windows 10
Sublime Text 3103

This plugin should be very useful to me. But i can get it work. I pressed '(' and nothing happens but only a ')' appears.

Any hints?

**Comment by pl-ca** [comment]
To be honest I haven't spent too much time on this plugin and is not fully stable yet. I'll need to see how it works in Windows. Did you manage to install ctags and generate a .tags file containing function definitions?

**Comment by canfoderiskii** [comment]
@pl-ca of course, I have ctags and I generated .tags and .tags_sorted_by_file.
